sentry/sentry.conf.example.py

```diff
@@ -213,8 +216,8 @@ def get_internal_network():
 # Web Server #
 ##############
 
-SENTRY_WEB_HOST = "0.0.0.0"
-SENTRY_WEB_PORT = 9000
+SENTRY_WEB_HOST = env("SENTRY_WEB_HOST", "0.0.0.0")
+SENTRY_WEB_PORT = env("SENTRY_WEB_PORT", 9000)
```

Comment:
We should almost never be modifying these as they don't do what you expect with a docker-compose setup.